blocks/library/button/style.scss

```diff
@@ -27,6 +27,10 @@ $blocks-button__height: 46px;
 		left: 50%;
 		transform: translateX( -50% );
 	}
+
+	.blocks-editable__tinymce[data-is-empty="true"]:before {
```

Comment:
Should this be so specific to button, or should we set this as the default for all empty Editable placeholders?

https://github.com/WordPress/gutenberg/blob/cab8713/blocks/editable/style.scss#L49